meet_balls_app/views.py

```diff
@@ -27,3 +34,43 @@ def loginUser(request):
 def logoutUser(request):
     logout(request)
     return render(request, 'meet_balls_app/home.html', {'user': {'is_authenticated': False}})
+
+
+@login_required(login_url="/login/")
+def profile(request, id):
+    player = Player.objects.get(user=id)
+    print(player.user.id, id)
```

Comment:
Do you need this print?

Reply:
Nope, it was needed for debugging purposes, and I forgot to remove it